styleguide/_layouts/default.jade

```diff
@@ -18,7 +18,7 @@ html(lang="en")
     link(href="#{ fonts }", rel="stylesheet")
     link(href="css/main.css", rel="stylesheet")
     link(href="https://fonts.googleapis.com/css?family=Open+Sans:400,600", rel="stylesheet")
-    link(href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.1/css/font-awesome.min.css" rel="stylesheet")
+    link(href="https://fonts.googleapis.com/css?family=Khand" rel="stylesheet")
```

Comment:
Is it just me, or does Khand looks strange now?

Before: 
![screen shot 2017-05-03 at 11 29 17 pm](https://cloud.githubusercontent.com/assets/5263371/25690595/5f37064c-3058-11e7-81c5-f91871f6c955.png)

After:
![screen shot 2017-05-03 at 11 29 13 pm](https://cloud.githubusercontent.com/assets/5263371/25690596/5f3a8fe2-3058-11e7-947e-c2a25f10a849.png)

Maybe google just renders it differently or something, but this is definitely a big difference here that I want to make sure we're aware of, as it may create discrepancies between www and zoolander.